app/templates/provider/setup.hbs

```diff
@@ -0,0 +1,48 @@
+<div class="moderation-setup">
+    <header>
+        <div class="container">
+            <div class="row">
+                <div class="text-center col-xs-12">
+                    <h1>{{t "setup.choose_settings" provider=model.name}}</h1>
+                </div>
+            </div>
+        </div>
+    </header>
+
+    <div class="text-bigger alert alert-warning text-center">
+        <i class="fa fa-exclamation-triangle" aria-hidden="true"></i>
+        {{t "setup.once_finalized"}}
+    </div>
+
+    <div class="container">
+        <div class="row m-t-lg">
+            <form class="col-md-offset-2 col-md-8 col-sm-offset-1 col-sm-10">
+                {{#each settingsOptions as |option|}}
+                    {{#form-block title=(t (concat "setup.settings." option.name ".title")) description=(t (concat "setup.settings." option.name ".description")) disabled=option.disabled}}
```

Comment:
Could the translation strings be computed in the controller instead of the template? It's a little hard to read but I don't feel strongly either way.